profiles/opentelemetry_dev/s6-rc-modifications/pulpcore-api/run

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/execlineb -S0
+s6-setuidgid pulp
+foreground {
+  /database/assets/wait_on_database_migrations.sh
+}
+foreground {
+  export DJANGO_SETTINGS_MODULE pulpcore.app.settings
+  export PULP_CONTENT_ORIGIN localhost
+  /usr/local/bin/pulpcore-manager collectstatic --clear --noinput --link
+}
+export DJANGO_SETTINGS_MODULE pulpcore.app.settings
+export PULP_SETTINGS /etc/pulp/settings.py
+export HOME /var/lib/pulp/
+
+# TODO: Modify OCI base image to allow for auto reload to be configurable so we don't have to override this.
+/usr/local/bin/gunicorn pulpcore.app.wsgi:application --bind "127.0.0.1:24817" --name pulp-api --access-logfile - --access-logformat "pulp [%({correlation-id}o)s]: %(h)s %(l)s %(u)s %(t)s \"%(r)s\" %(s)s %(b)s \"%(f)s\" \"%(a)s\"" --reload
```

Comment:
This would be prepended with `/usr/local/bin/opentelemetry-instrument --service_name pulp-api`